spaces/clover/index.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Clover",
+  "chainId": 1,
+  "decimals": 6,
+  "symbol": "CLV",
+  "defaultView": "all",
+  "token": "0x22222C03318440305aC3e8a7820563d6A9FD777F",
+  "core": [
+  ],
+  "min": 22,
+  "invalid": [],
+  "skin": "example"
```

If you use the default theme you will want to remove the skin field.
```suggestion
  "invalid": []
```